bindings/netframework/ElectionGuard.NetFramework/ElectionGuardInterop/Group.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
+
+#include "constants.h"
+#include "group.hpp"
+#include "ManagedInstance.h"
+using namespace System;
+using namespace System::Runtime::InteropServices;
+
+namespace ElectionGuardInterop
+{
+  public
+    ref class ElementModP : ManagedInstance<electionguard::ElementModP>
+    {
+      public:
+        ElementModP(array<uint64_t> ^ elem)
+        {
+            pin_ptr<uint64_t> pinned = &elem[0];
+            uint64_t array[MAX_P_LEN]{};
+            for (int i = 0; i < MAX_P_LEN; i++) {
+
+                array[i] = pinned[i];
+            }
+            this->_instance = new electionguard::ElementModP(array);
+        }
+      internal: 
+        ElementModP(electionguard::ElementModP *elem) : ManagedInstance(elem, false)
```

Comment:
basically saying that any internal calls to the library like this are simply facade pointers that dont own the underlying resource.